{"name": "Логин под существующим пользователем", "status": "passed", "description": "Проверка успешного логина под существующим пользователем с валидными данными.", "start": 1726233882158, "stop": 1726233882453, "uuid": "a6ba3645-f485-408b-afab-0dcb3b496dfb", "historyId": "46b44e11f632d8becaea359ceff43819", "testCaseId": "46b44e11f632d8becaea359ceff43819", "fullName": "tests.test_user_login.TestUserLogin#test_login_with_existing_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "HOME-PC"}, {"name": "thread", "value": "12588-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}